proposals/0199-strides-revamp.md

```diff
@@ -0,0 +1,220 @@
+# Revamp the StrideTo/StrideThrough types
+
+* Proposal: [SE-0199](0199-strides-revamp.md)
+* Authors: [Max Moiseev](https://github.com/moiseev), [Xiaodi Wu](https://github.com/xwu)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+
+* Implementation: [apple/swift#14288](https://github.com/apple/swift/pull/14288)
+
+<!--
+
+* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution/), [Additional Commentary](https://lists.swift.org/pipermail/swift-evolution/)
+* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
+* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal: [SE-XXXX](XXXX-filename.md)
+
+-->
+
+## Introduction
+
+The Swift standard library provides two sequence types `StrideTo` and
+`StrideThrough`, instances of which are returned from `stride(from:to:by:)`
+and `stride(from:through:by:)` respectively. These types are designed to be more
+flexible versions of Swift ranges, allowing start values less than end values
+and stride values other than 1.
```

Comment:
Don't you mean "start values _greater_ than end values"?

Reply:
Most likely.